model/timespan.go

```diff
@@ -12,6 +12,7 @@ type TimeSpan struct {
 	OffsetUTC     int
 	UserID        int `gorm:"type:int REFERENCES users(id) ON DELETE CASCADE"`
 	Tags          []TimeSpanTag
+	Notes         *string
```

Comment:
Can we make this not nullable? In the UI we don't need to differentiate between nil and empty string.

Reply:
Also not nullable in the graphql types?

Reply:
Yes.